### ISSUE TITLE
Unclear where to put .love file

### ISSUE THREAD

**Issue by ohiogauze** [ahrnbom/l-veftw#2]
Hi! I'm currently messing with this repo but I'm not sure how to add a .love file in order for it to be loaded. Could I possibly get instructions on how to do this? Thank you!

**Comment by ahrnbom** [comment]
It's not worth it, this repo just doesn't work. If you want Löve on UWP, I think you're better off starting from scratch.
________________________________

**Comment by ohiogauze** [comment]
Hmm. I can get LÖVE running fine with a "no game" screen using this repo
though. Is there no way to make it load a .love on startup at all?

**Comment by ahrnbom** [comment]
Yeah, there is. But it doesn't seem to work, far too many things aren't working for it to be worth it, that's what I mean. If you care about getting any real game running, you're going to have to redo and fix a lot of things. It's probably not worth it, I think you're better off starting from scratch with the most recent version of Löve, perhaps reusing some of the dependencies that I've ported to UWP.
________________________________

**Comment by ohiogauze** [comment]
Damn, okay thank you!

**Comment by oechsler** [comment]
I know that this project has been discontinued. Nevertheless there is a way to sort of bodge this in by just forcing love to load a .love file via argc and argv. This is the way its done in the original LöveFTW for 8.1.

```cpp
// Since we don't have access to the actual argc and argv 
// from SDL's main function, we'll just hack them inside here
argc = 2;
char* lovefile = "game.love";
char* args[2] = {"love.exe", lovefile};
argv = args;
```
